src/Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X/RazorExtensions.cs

```diff
@@ -33,6 +33,19 @@ public static void Register(IRazorEngineBuilder builder)
             builder.Features.Add(new MvcViewDocumentClassifierPass());
         }
 
+        public static void Register(RazorProjectEngineBuilder builder)
+        {
+            if (builder == null)
+            {
+                throw new ArgumentNullException(nameof(builder));
+            }
+
+            builder.SetImportFileName("_ViewImports.cshtml");
```

Comment:
This should probably be part of the imports feature. As in, we get rid of the concept of 'ImportsFileName'. You want imports, you figure it out 👍